src/sentry/sentry_apps/api/endpoints/installation_external_requests.py

```diff
@@ -40,18 +38,6 @@ def get(self, request: Request, installation) -> Response:
         if project:
             kwargs.update({"project_slug": project.slug})
 
-        try:
-            choices = SelectRequester(**kwargs).run()
-        except ValidationError as e:
-            return Response(
-                {"error": e.message},
-                status=400,
-            )
-        except APIError:
-            message = f'Error retrieving select field options from {request.GET.get("uri")}'
-            return Response(
-                {"error": message},
-                status=400,
-            )
```

Comment:
same here, any reason we're not capturing these anymore?

Reply:
same as above